scripts/annotate.py

```diff
@@ -4,7 +4,7 @@
 from pycytominer.annotate import annotate
 
 
-def annotate_cells(aggregated_data, barcodes_path):
+def annotate_cells(aggregated_data, barcodes_path, output, configs):
```

Comment:
`configs` is an argument in `annotate.py` but not `aggregate.py`? Just checking that this is intentional. (It might confuse users to see configs in one but not the other)

Reply:
This was a mistake on my end, `configs` should be a new parameter to all functions.